src/common/components/signUpActivationModal/signUpActivationModal.tpl.html

```diff
@@ -89,7 +89,7 @@ <h4 translate>Your email has been activated!</h4>
             </div>
           </div>
 
-          <div class="row active-account">
+          <div class="row active-account" style="margin-bottom: 20px">
```

Comment:
Seems this margin should be added to the styles for `active-account` instead of being inline.

Reply:
Good catch. I've fixed it.